src/main.ts

```diff
@@ -23,8 +24,8 @@ pinia.use(({ store }) => {
 	const state = cloneDeep(store.$state)
 	store.$reset = () => store.$patch(state)
 })
-
 app.use(pinia)
 app.use(router)
+app.directive('vLock', vLock)
```

Comment:
The directive name should be just `lock`, Vue already adds the `v-` prefix when declared globally